dotcom-rendering/src/frontend/feFootballDataPage.ts

```diff
@@ -0,0 +1,44 @@
+import type { EditionId } from '../lib/edition';
+import type { FooterType } from '../types/footer';
+import type { FENavType } from '../types/frontend';
+import type { FEFrontConfig } from './feFront';
+
+export type FERound = {
+	roundNumber: string;
+	name?: string;
+};
+
+export type FECompetitionSummary = {
+	id: string;
+	url: string;
+	fullName: string;
+	nation: string;
+	dividers?: number[];
+};
+
+export type FEFootballCompetition = {
+	name: string;
+	url: string;
+};
+
+export type FEFootballPageConfig = Omit<
+	FEFrontConfig,
+	'keywordIds' | 'keywords' | 'isFront'
+> & {
+	isFront: boolean;
+	hasSurveyAd: boolean;
+};
+
+export type FEFootballDataPage = {
+	nextPage?: string;
+	previousPage?: string;
```

Comment:
I think these only exist on the match list page? See https://github.com/guardian/frontend/pull/27845